CoNet/Main/Home/CalendarDateFormatter.swift

```diff
@@ -8,33 +8,67 @@
 import Foundation
 
 class CalendarDateFormatter {
-    private let calendar = Calendar.current // Calendar 구조체를 현재 달력으로 초기화
+    private var calendar = Calendar.current // Calendar 구조체를 현재 달력으로 초기화
     private let dateFormatter = DateFormatter() // 원하는 String 타입으로 변화시켜줄 formatter
     private var nowCalendarDate = Date() // 현재 시간
     private(set) var days = [String]() // 달력에 표시할 날짜를 담을 배열
     
     init() {
         dateFormatter.locale = Locale(identifier: "ko_KR")
         dateFormatter.timeZone = TimeZone(abbreviation: "KST")
+        calendar.locale = Locale(identifier: "ko_KR")
+        calendar.timeZone = TimeZone(identifier: "Asia/Seoul")!
         
         configureCalendar()
     }
     
+    // 오늘 날짜 Int 배열 
+    func getToday() -> DateComponents {
+        let calendarInfo = calendar.dateComponents([.year, .month, .day], from: Date())
+        
+        return calendarInfo
+    }
+    
+    // 캘린더 날짜 Int 배열
+    func getDate() -> [Int] {
+        let calendarInfo = calendar.dateComponents([.year, .month, .day], from: nowCalendarDate)
+        
+        return [calendarInfo.year!, calendarInfo.month!, calendarInfo.day!]
+    }
```

Comment:
날짜를 [Int]로 받아오는건 주로 날짜를 받아서 String으로 변환하기 때문인가여??
Date나 DateComponent(?) 타입으로 가지고 있지 않고 Int로 return하는 이유가 궁금해여!

Reply:
CalendarViewController 파일에서 초기화를 할 때 `var date: [Int] = []` 와 같이 선언할 수 있는데 
DateComponent 타입의 경우 `var date: DateComponent = ?` 저 물음표 부분에 무엇이 들어가야할지,, 이런 문제 때문에 [Int]로 선언했습니다

Reply:
아아 그러면 지금 nowCalendarDate가 Date()라서? getToday와 동일한데 getDate는 [Int]로 받아오는거니까
```suggestion
    func getTodayIntArray() -> [Int] {
        let today = getToday()
        return [today.year!, today.month!, today.day!]
    }
```
근데 이름이 마땅히 생각이 안나지만 이렇게 해도 좋을 것 같아여! 
그리고 force unwrapping을 최대한 안쓰면 좋을 것 같긴한데.. 아무튼 그렇습니당!
그냥 내 생각! ㅎㅎ

Reply:
아아 넹! 그리고 nowCalendarDate는 달 이동에 따라 바뀌기 때문에 오늘의 날짜가 아닌 보여지는 캘린더 화면의 날짜입니다! 
getCalendarDateIntArray 이런 식으로 바꿔볼까요?? 
force unwrapping은 .. 다른 아이디어가 있다면 알려주세요~

Reply:
오호 네넹ㅎㅎ 쪼아용
흠 흠 고민해볼게요 으하하